docs/releases/1.9.2.txt

```diff
@@ -71,3 +71,7 @@ Bugfixes
 
 * Fixed a crash when using a reverse ``OneToOneField`` in
   ``ModelAdmin.readonly_fields`` (:ticket:`26060`).
+
+* Fixed a crash when calling the ``migrate`` command in a test case with the
```

Comment:
Interesting... it looks to me like there's no need to call `migrate` in that test anyway -- could be replaced by calling `create_permissions()` directly. Anyway, feel free to proceed with this.